kolibri/tasks/api.py

```diff
@@ -190,6 +191,11 @@ def localdrive(self, request):
 
         return Response(out)
 
+    @list_route(methods=['get'])
```

Comment:
This seems like it might be better put somewhere other than on the TasksViewSet, it doesn't seem semantically related to me?

Reply:
Yeah, I think we need to move it somewhere else. Maybe when we have a channel detail page, we can move it there, since it's going to be used in that page?

Reply:
In general I think we should have a 'devices' module, which contains all the operations involving the system where kolibri is installed. We should also move the `usbdiscover` module there as well.

Maybe @lyw07 you can create a kolibri/localdevice module, then put this endpoint there?

Reply:
fyi, @benjaoming added `kolibri.core.deviceadmin`. not sure if it belongs there?

Reply:
@aronasorman yes sure, I can do that. But what is the `usbdiscover` module you are talking about? Sorry I'm not following you here. 
@indirectlylit It seems that `deviceadmin` is for database backup? Not sure if I can put it there..

I was thinking previously that putting this endpoint to the `content` module under ChannelMetadataViewset, since we only call this in the channel detail page, but may not be a good idea?